src/node_options.cc

```diff
@@ -1216,6 +1216,9 @@ PerProcessOptionsParser::PerProcessOptionsParser(
   AddOption("--run",
             "Run a script specified in package.json",
             &PerProcessOptions::run);
+  AddOption("--run-dir",
```

Comment:
I've seen that this has already been raised. While I don't have a better name in mind, I think the current name is misleading

Reply:
I agree with @GeoffreyBooth  suggestion 

```--run-path`` +1

Reply:
I will wait for more advice to be given, please feel free to give feedback 🚀

Reply:
@mertcanaltin , IMHO the most common term used in such context is `working_directory` or `working_dir`.
E.g. "cwd" stands for "current working directory", "pwd" - "print working directory", etc.

Another example is the `WORKING_DIRECTORY` attribute used by [CMake](https://cmake.org/cmake/help/latest/command/add_custom_target.html). 

The new code would read: `cwd = node::per_process::cli_options->working_dir;`
I believe it is more natural than `cwd = node::per_process::cli_options->run_dir;`

Reply:
The `--cwd` is also a great name proposed above by @targos

Reply:
Please disregard my previous posts. I read through the discussion in the linked issue and see why the "working directory" term is not used. I am actually puzzled why the new flag is needed at all. Users can always provide the full path for the `--run` parameter.

Reply:
@vmoroz, thank you very much for your suggestions. I think adding this flag gives users an easy and clear way to specify the correct directory and ensures that commands work in the intended context. This extra flexibility can provide a good experience in scenarios where precise control over the working directory is needed, such as when running new API documentation tools.

Reply:
May I suggest `--run-from`

Reply:
As far as I know this argument is supposed to indicate where the package.json is located at, such as a CWD, it is supposed to be like the CLI flag that exists within npm. But maybe Im mistaken here? 🤔